lib/guardian.rb

```diff
@@ -622,4 +621,18 @@ def can_do?(action, obj)
     end
   end
 
+  protected
+
+  def category_group_moderation_enabled?
+    return false if !authenticated?
+    return false if !SiteSetting.enable_category_group_moderation
+    true
+  end
```

Comment:
This method may return `false`, even if category group moderation is enabled, so the method name seems wrong.

Maybe a better name would be something like `has_category_moderator_permissions?` or similar?

Reply:
Good called. I changed it to `category_group_moderation_allowed?`.